Nodejs/Product/Nodejs/ProjectTemplates/AzureExpress4App/package.json

```diff
@@ -20,5 +20,10 @@
   },
   "engines": {
     "node": "~6.10.x"
+  },
+  "devDependencies": {
+    "eslint": "^8.21.0"
+  },
+  "eslintConfig": {
```

Comment:
What does an empty `eslintConfig` field means?

Reply:
It means that we're explicitly defining an ESLint configuration (which will allow the server to pick it up and not prompt the user to run the plugin wizard) but not indicating the rules to use.